src/textarea.test.jsx

```diff
@@ -93,6 +97,25 @@ describe('Textarea', function() {
 		const boundComponent = TestUtils.renderIntoDocument(<Textarea
 			name={NAME_ATTR}
 			label={LABEL_TEXT}
+			id={ID}
+			value={VALUE} />);
+
+		textareaEl = TestUtils.findRenderedDOMComponentWithTag(boundComponent, 'textarea');
+		TestUtils.Simulate.change(textareaEl, { target: { value: newValue } });
+
+		expect(changeSpy).toHaveBeenCalled();
+		expect(stateSpy).toHaveBeenCalledWith({ value: newValue });
+	});
+	it('should handle onChange as a prop', function() {
+		const newValue = `${VALUE}r`;
+		const changeSpy = spyOn(Textarea.prototype, 'onChange').and.callThrough();
+		const stateSpy = spyOn(Textarea.prototype, 'setState').and.callThrough();
+
+		const boundComponent = TestUtils.renderIntoDocument(<Textarea
```

Comment:
and to be obnoxiously nitpicky, can we move the `<Textarea` to a new line, it is harder to read when it's on the same line with the renderIntoDoc